### PR TITLE
client/webserver: Update Order-details Type to show the correct value.

### PR DESCRIPTION
**Pull request by dev-warrior777**
Currently the order details Type is showing
	market.quoteUnitInfo.conventional.unit

	Also cleaned up walletpair BW_ARGS to override
	dexc.conf options.

### CODE REVIEW

**Comment by dev-warrior777** [comment]
Hopefully can resolve [#3044](https://github.com/decred/dcrdex/issues/3043#issue-2615762620)

**Comment by dev-warrior777** [comment]
Current conflicts are in walletpair which is just a suggested upgrade to `BW_ARGS`

File: client/webserver/site/src/js/markets.ts [hunk 1]
```diff
@@ -1703,7 +1703,10 @@ export default class MarketsPage extends BasePage {
       else rateStr = Doc.formatRateFullPrecision(ord.rate, market.baseUnitInfo, market.quoteUnitInfo, cfg.ratestep)
       details.rate.textContent = mord.header.rate.textContent = rateStr
       header.baseSymbol.textContent = market.baseUnitInfo.conventional.unit
-      details.type.textContent = market.quoteUnitInfo.conventional.unit
+
+      details.type.textContent = 'Fred'
```
Comment (ukane-philemon) [inline client/webserver/site/src/js/markets.ts:1707]: testing?...

**Review by ukane-philemon — CHANGES_REQUESTED** [review]

File: client/webserver/site/src/js/markets.ts (hunk 1 above)
Comment (dev-warrior777) [reply client/webserver/site/src/js/markets.ts:1707]: yup .. now I need to find the right replacement variable for the order type .. my first time in the UI code

**Comment by dev-warrior777** [comment]
Should `limit`, `market`, `cancel` be localized ..? Chinese, Arabic, currently look like this ..

chinese
arabic

..or are these terms understood/used by traders?

**Comment by buck54321** [comment]
See e.g. `intl.prep(intl.ID_MARKET_ORDER)`

**Comment by dev-warrior777** [comment]
TODO:

- Some locales still default to locale en_US but it is the correct order type.
- There is not yet any translation for order type cancel for any locale
- Arabic seems to have the most translations for order type

Since UI details get noticed we may want to put the above into the point release as is. I will work on missing translations in another PR.

**Comment by buck54321** [comment]
How about this? https://github.com/buck54321/dcrdex/compare/017099c...buck54321:dcrdex:user-order-type-intl

**Comment by dev-warrior777** [comment]
> How about this? [buck54321/dcrdex@017099c...buck54321:dcrdex:user-order-type-intl](https://github.com/buck54321/dcrdex/compare/017099c...buck54321:dcrdex:user-order-type-intl)

That is much more complete .. I will update using your approach of a new OrderUtil util, thanks

**Review by buck54321 — APPROVED** [review]

**Pull request merged**